packages/svelte/src/internal/client/dev/debug.js

```diff
@@ -103,7 +114,13 @@ function log_dep(dep) {
 		const derived = /** @type {Derived} */ (dep);
 
 		// eslint-disable-next-line no-console
-		console.groupCollapsed('%cderived', 'font-weight: normal', derived.v);
+		console.groupCollapsed(
+			`%c$derived %c${dep.label ?? '<unknown>'}`,
+			'font-weight: bold; color: CornflowerBlue',
+			'font-weight: normal',
+			snapshot(derived.v)
```

Comment:
should be untracked to avoid adding an accidental dependency when logging in a reactive context